.travis.yml

```diff
@@ -1,35 +1,54 @@
+os: linux
 dist: bionic
-sudo: required
-language:
-  - c
+language: c
 compiler:
   - gcc
 addons:
   apt:
     sources:
       - ubuntu-toolchain-r-test
     packages:
-      - cmake
+      - cmake cppcheck doxygen lcov graphviz
 env:
   global:
    - SIMULATION=native
    - ENABLE_UNIT_TESTS=true
-  matrix:
+   - DEPLOY_DIR=$TRAVIS_BUILD_DIR/deploy
+  jobs:
    - BUILDTYPE=release OMIT_DEPRECATED=true
    - BUILDTYPE=release OMIT_DEPRECATED=false
    - BUILDTYPE=debug OMIT_DEPRECATED=true
    - BUILDTYPE=debug OMIT_DEPRECATED=false
 
-before_install:
-  - sudo apt-get install cppcheck doxygen lcov graphviz
+# Build the pdfs and copy to the deploy directory
+before_deploy:
+ - sudo apt-get install texlive-latex-base texlive-fonts-recommended texlive-fonts-extra texlive-latex-extra
+ - mkdir $DEPLOY_DIR
+ - cd $TRAVIS_BUILD_DIR/build/doc/users_guide/latex
+ - make > build.txt
```

Comment:
Should this also capture stderr?  (i.e. `2>&1` or similar)

What is `build.txt` used for after this - is it discarded?

Reply:
I haven't found any of the stdout messages very useful from the latex build, they just pollute the log (100's of them for using the packages I had installed locally)... but I wasn't ready to go full /dev/null on them.  I don't do anything with them, but they are there for now if someone does want to.  I let stderr go to the log in case there is anything useful, but as long as the refman.pdf doesn't get built I didn't make them an error.

No real reasons other than a choice that worked and seemed reasonable.  Happy to change if there are other preferences.